styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,371 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
```

Comment:
I wanted to make sure that for code review purposes that everything was mostly all in one place and when I wrote this there was no bootstrap_overrides file. I will be sure to move all of that over once I address the rest of the concerns in here. Thank ya sir. I'll get with you on how you would prefer that file organized.